release/release_tests.yaml

```diff
@@ -5348,6 +5348,59 @@
         cluster_env: app_config.yaml
         cluster_compute: data_ingest_benchmark_compute_gce.yaml
 
+- name: streaming_data_ingest_benchmark_100gb_gpu
+  group: data-tests
+  working_dir: nightly_tests/dataset
+
+  frequency: nightly
+  team: data
+  cluster:
+    cluster_env: app_config.yaml
+    cluster_compute: data_ingest_benchmark_compute_gpu.yaml
+
+  run:
+    timeout: 300
+    script: python data_ingest_benchmark.py --dataset-size-gb=100 --num-workers=4 --new_streaming --use-gpu
+    wait_for_nodes:
+      num_nodes: 3
+
+  variations:
+    - __suffix__: aws
+    - __suffix__: gce
+      env: gce
+      frequency: manual
+      cluster:
+        cluster_env: app_config.yaml
+        cluster_compute: data_ingest_benchmark_compute_gpu_gce.yaml
+
+# This case will early stop the data ingestion iteration on the GPU actors.
+# This is a common usage in PyTorch. And there was a bug in Ray Data that caused
+# GPU memoy leak.
```

Comment:
> And there was a bug in Ray Data that caused GPU memoy leak.

There was a bug that cause GPU memory leaks when performing early stopping?

Reply:
@c21 knows more context. Is there an issue that I can link to here?